source/common/router/router.cc

```diff
@@ -1064,7 +1068,7 @@ void Filter::onPerTryTimeoutCommon(UpstreamRequest& upstream_request, Stats::Cou
 void Filter::onStreamMaxDurationReached(UpstreamRequest& upstream_request) {
   upstream_request.resetStream();
 
-  if (maybeRetryReset(Http::StreamResetReason::LocalReset, upstream_request)) {
+  if (maybeRetryReset(Http::StreamResetReason::LocalReset, upstream_request, /*reachPerTryTimeout*/ false)) {
```

Comment:
the comment casing is inconsistent w/ the actual variable name. also should this be `is_per_try_timeout` instead of `reach_per_try_timeout`?